apps/dialtone-documentation/docs/components/empty-states.md

```diff
@@ -0,0 +1,378 @@
+</ol>
+<br>
+There is multiple ways to use the empty state on variable size, here are some examples:
+
+### Small size with icon
+
+<code-well-header>
+<dt-empty-state
+  size="sm"
+  icon-name="concierge-bell"
+  header-text="Nothing to see here"
+  body-text="Looks like theres no data to display here."
+>
+  <template #body>
+    <div class="d-mt8 d-stack d-stack--row d-stack--gap-300">
+      <button class="d-btn">Click me</button>
+      <button class="d-btn d-btn--primary">Click me</button>
+    </div>
+  </template>
+</dt-empty-state>
+</code-well-header>
+
+<code-example-tabs
+htmlCode='
+<div class="d-empty-state d-empty-state--size-sm">
+  <svg>...</svg>
+  <div class="d-p16">
+    <h1>Nothing to see here</h1>
+    <p>Looks like theres no data to display here.</p>
+  </div>
+  <div class="d-mt8 d-stack d-stack--row d-stack--gap-300">
+    <button class="d-btn">Click me</button> 
+    <button class="d-btn d-btn--primary">Click me</button>
+  </div>
+</div>
+'
+vueCode='
+<dt-empty-state
+  size="sm"
+  icon-name="concierge-bell"
+  header-text="Nothing to see here"
+  body-text="Looks like theres no data to display here."
+>
+  <template #body>
+    <div class="d-mt8 d-stack d-stack--row d-stack--gap-300">
+      <button class="d-btn">Click me</button>
+      <button class="d-btn d-btn--primary">Click me</button>
+    </div>
+  </template>
+</dt-empty-state>
+'
+showHtmlWarning />
+
+### Medium size without illustration and icon
+
+<code-well-header>
+<dt-empty-state
+  size="md"
+  icon-name="concierge-bell"
+  header-text="Nothing to see here"
+  body-text="Looks like theres no data to display here."
+>
+  <template #body>
+    <div class="d-mt8 d-stack d-stack--row d-stack--gap-300">
+      <button class="d-btn">Click me</button>
+      <button class="d-btn d-btn--primary">Click me</button>
+    </div>
+  </template>
+</dt-empty-state>
+</code-well-header>
+
+<code-example-tabs
+htmlCode='
+<div class="d-empty-state d-empty-state--size-md">
+  <svg>...</svg>
+  <div class="d-p16">
+    <h1>Nothing to see here</h1>
+    <p>Looks like theres no data to display here.</p>
+  </div>
+  <div class="d-mt8 d-stack d-stack--row d-stack--gap-300">
+    <button class="d-btn">Click me</button> 
+    <button class="d-btn d-btn--primary">Click me</button>
+  </div>
+</div>
+'
+vueCode='
+<dt-empty-state
+  size="md"
+  icon-name="concierge-bell"
+  header-text="Nothing to see here"
+  body-text="Looks like theres no data to display here."
+>
+  <template #body>
+    <div class="d-mt8 d-stack d-stack--row d-stack--gap-300">
+      <button class="d-btn">Click me</button>
+      <button class="d-btn d-btn--primary">Click me</button>
+    </div>
+  </template>
+</dt-empty-state>
+'
+showHtmlWarning />
+
+## Vue API
+
+<component-vue-api component-name="empty-state" />
+
+## Classes
+
+At minimum, empty state contains body of content. It could also have header with buttons, and footer with buttons/text.
```

Comment:
At a minimum, empty state requires a body of content. It can optionally contain a header with buttons, or a footer with buttons/text.